tests/test_data_stats.py

```diff
@@ -166,6 +179,18 @@ def test_file(self, input_data, expected_print):
             if sys.platform != "win32":
                 self.assertEqual(content, expected_print)
 
+    def test_multiple_data_stats(self):
```

Comment:
there's some issue with this test
```
[2022-07-01T17:05:50.251Z] ======================================================================
[2022-07-01T17:05:50.251Z] FAIL: test_multiple_data_stats (tests.test_data_stats.TestDataStats)
[2022-07-01T17:05:50.251Z] ----------------------------------------------------------------------
[2022-07-01T17:05:50.251Z] Traceback (most recent call last):
[2022-07-01T17:05:50.251Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_data_stats.py", line 190, in test_multiple_data_stats
[2022-07-01T17:05:50.251Z]     self.assertEqual(
[2022-07-01T17:05:50.251Z] AssertionError: '' != "Data statistics:\nType: <class 'numpy.nd[45 chars], 2)"
[2022-07-01T17:05:50.251Z] + Data statistics:
[2022-07-01T17:05:50.251Z] + Type: <class 'numpy.ndarray'> int64
[2022-07-01T17:05:50.251Z] + Shape: (2, 2)
[2022-07-01T17:05:50.251Z] + Value range: (0, 2)
[2022-07-01T17:05:50.251Z] 
[2022-07-01T17:05:50.251Z] ----------------------------------------------------------------------
[2022-07-01T17:05:50.251Z] Ran 7228 tests in 1194.812s
```

```
[2022-07-01T16:39:58.785Z] + python -c 'import monai; monai.config.print_debug_info()'
[2022-07-01T16:40:02.067Z] ================================
[2022-07-01T16:40:02.069Z] Printing MONAI config...
[2022-07-01T16:40:02.069Z] ================================
[2022-07-01T16:40:02.069Z] MONAI version: 0+unknown
[2022-07-01T16:40:02.069Z] Numpy version: 1.21.2
[2022-07-01T16:40:02.069Z] Pytorch version: 1.10.0a0+0aef44c
[2022-07-01T16:40:02.069Z] MONAI flags: HAS_EXT = False, USE_COMPILED = False
[2022-07-01T16:40:02.069Z] MONAI rev id: None
[2022-07-01T16:40:02.069Z] MONAI __file__: /home/jenkins/agent/workspace/MONAI-premerge/monai/monai/__init__.py
[2022-07-01T16:40:02.069Z] 
[2022-07-01T16:40:02.069Z] Optional dependencies:
[2022-07-01T16:40:02.069Z] Pytorch Ignite version: 0.4.9
[2022-07-01T16:40:02.069Z] Nibabel version: 4.0.1
[2022-07-01T16:40:02.069Z] scikit-image version: 0.19.3
[2022-07-01T16:40:02.069Z] Pillow version: 9.1.1
[2022-07-01T16:40:02.069Z] Tensorboard version: 2.6.0
[2022-07-01T16:40:02.069Z] gdown version: 4.5.1
[2022-07-01T16:40:02.069Z] TorchVision version: 0.11.0a0
[2022-07-01T16:40:02.069Z] tqdm version: 4.62.3
[2022-07-01T16:40:02.069Z] lmdb version: 1.2.1
[2022-07-01T16:40:02.069Z] psutil version: 5.8.0
[2022-07-01T16:40:02.069Z] pandas version: 1.4.3
[2022-07-01T16:40:02.069Z] einops version: 0.4.1
[2022-07-01T16:40:02.069Z] transformers version: 4.20.1
[2022-07-01T16:40:02.069Z] mlflow version: 1.27.0
[2022-07-01T16:40:02.069Z] pynrrd version: 0.4.3
[2022-07-01T16:40:02.069Z] 
[2022-07-01T16:40:02.069Z] For details about installing the optional dependencies, please visit:
[2022-07-01T16:40:02.069Z]     https://docs.monai.io/en/latest/installation.html#installing-the-recommended-dependencies
```

Reply:
I believe this is due to the limitation of jenkins platform, I am not sure if it supports stdout redirecting or not. I have changed to use `unittest.mock`, let's see if it works.

Reply:
thanks, the github ci already confirms it's usage, let me change it to a trivial print test.